src/commons/utils/corio_utils.py

```diff
@@ -384,3 +397,17 @@ def list_dirs(self, remote_path: str) -> list:
             return []
         finally:
             self.disconnect()
+
+    def is_package_installed(self, package_name):
+        """
+        check package is installed or not.
+
+        :param package_name: package name to check
+        """
+        self.connect()
+        resp = self.execute_command("rpm -qa | grep {}".format(package_name))
```

Comment:
keep command in commands.py and then use it here would help if we need it for different package.

Reply:
moved commands to commands.py